python/mxnet/initializer.py

```diff
@@ -438,25 +483,32 @@ class FusedRNN(Initializer):
     bidirectional : bool
         should be the same with arguments passed to FusedRNNCell.
     """
-    def __init__(self, init, num_hidden, num_layers, mode, bidirectional=False):
+    def __init__(self, init, num_hidden, num_layers, forget_bias, mode, bidirectional=False):
         if not isinstance(init, Initializer):
             klass, kwargs = json.loads(init)
             init = _INITIALIZER_REGISTRY[klass.lower()](**kwargs)
         super(FusedRNN, self).__init__(init=init.dumps(), num_hidden=num_hidden,
-                                       num_layers=num_layers, mode=mode,
-                                       bidirectional=bidirectional)
+                                       num_layers=num_layers, forget_bias=forget_bias,
+                                       mode=mode, bidirectional=bidirectional)
         self._num_hidden = num_hidden
         self._num_layers = num_layers
+        self._forget_bias = forget_bias
         self._bidirectional = bidirectional
         self._mode = mode
         self._init = init
 
     def _init_weight(self, _, arr):
         from .rnn import rnn_cell
-        cell = rnn_cell.FusedRNNCell(self._num_hidden, self._num_layers,
+        cell = rnn_cell.FusedRNNCell(self._num_hidden, self._num_layers, self._forget_bias,
                                      self._mode, self._bidirectional, prefix='')
         args = cell.unpack_weights({'parameters': arr})
         for name in args:
             desc = InitDesc(name)
-            self._init(desc, args[name])
+            # for lstm bias, we use a custom initializer
+            # which adds a bias to the forget gate
+            if self._mode == 'lstm':
+                FusedLSTMBias(forget_bias=self._forget_bias)._init_weight(desc, args[name])
```

Comment:
I have removed it. 
Is anyone working on adding simple regression tests on RNN? I have only seen regression test for conv net, we could check an error metric on a simple datasets training with GRU/LSTM.